modules/juce_graphics/native/juce_mac_CoreGraphicsContext.mm

```diff
@@ -954,6 +954,26 @@ Image juce_createImageFromUIImage (UIImage* img)
 #endif
 
 #if JUCE_MAC
+ Colour juce_convertColourToDisplayColourSpace (const Colour& colour,
+                                         const void* screenPtr)
+ {
+     auto srgbSpace = CGColorSpaceCreateWithName (kCGColorSpaceSRGB);

```

Comment:
In case creating and releasing the color space object is a bit expensive, perhaps the SRGB color space could be a singleton?

Reply:
I thought about it. it seems not expensive. so we can do a fancy struct 
ColourSpace just to release it (CGColorSpaceRefRelease). So is it worth pre-optimizing it?

Reply:
Ok we can live without it and then if we notice it in profiling consider doing something

Reply:
Anyway I've made a singleton and it works not sure how much performance gain. but a little DRYer code. (though more lines :) )